cloudigrade/api/models.py

```diff
@@ -92,12 +95,19 @@ def __str__(self):
         """Return the friendly string representation for the Api User."""
         return (
             f"{self.__class__.__name__}("
+            f"id={self.id}, "
```

Comment:
Want to add `is_permanent` here too, or was it an intentional omission?

Reply:
👍 no problem I'll add it. we'll also add date_joined for consistency with the InternalUserSerializer.